ts/input/tex/amscd/AmsCdMethods.ts

```diff
@@ -124,37 +124,35 @@ const AmsCdMethods: { [key: string]: ParseMethod } = {
         if (!a) {
           a = '\\kern ' + top.getProperty('minw');
         } // minsize needs work
-        if (a || b) {
-          const pad: EnvList = { width: '+.67em', lspace: '.33em' };
-          mml = parser.create('node', 'munderover', [mml]) as MmlMunderover;
-          if (a) {
-            const nodeA = new TexParser(
-              a,
-              parser.stack.env,
-              parser.configuration
-            ).mml();
-            const mpadded = parser.create('node', 'mpadded', [nodeA], pad);
-            NodeUtil.setAttribute(mpadded, 'voffset', '.1em');
-            NodeUtil.setChild(mml, mml.over, mpadded);
-          }
-          if (b) {
-            const nodeB = new TexParser(
-              b,
-              parser.stack.env,
-              parser.configuration
-            ).mml();
-            NodeUtil.setChild(
-              mml,
-              mml.under,
-              parser.create('node', 'mpadded', [nodeB], pad)
-            );
-          }
-          if (parser.configuration.options.amscd.hideHorizontalLabels) {
-            mml = parser.create('node', 'mpadded', mml, {
-              depth: 0,
-              height: '.67em',
-            });
-          }
+        const pad: EnvList = { width: '+.67em', lspace: '.33em' };
+        mml = parser.create('node', 'munderover', [mml]) as MmlMunderover;
+        if (a) {
```

Comment:
So why do we need that `if` here if we already know that `a` is always set?

Reply:
True enough.  I didn't notice it, because I was just responding to the fact that the `|| b` portion of the `if statement above was not being covered, and didn't look further to see if anything else could be eliminated.

Reply:
I've made the change.